src/Label/Commands/ExcludeLabel.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Label\Commands;
+
+final class ExcludeLabel extends AbstractCommand
```

Comment:
Other commands don't have the `Label` suffix. So `Exclude` could be enough?

Reply:
Problem is that `Include` is a reserved keyword in PHP, so you cannot have a class `Include` `Exclude` is a valid classname but that would make it inconsitent with `Include`.